decisions/versioning-rules.md

```diff
@@ -0,0 +1,32 @@
+<!-- Copy this to make your own decisions. -->
+
+# Versioning approach for component releases
+
+## TL;DR
+<!-- Only if long enough to merit one. -->
+
+## Decision
+When a new Backpack component is to be released the approach is to use semver to manage versions which are based on the types of changes that have been made to each component. To ensure that changes made don't cause unexpected outages or issues throughout the applications the following reasoning should be followed.
+
+### Major
+For Major releases these would be changes that do the following:
+- If changes are not backwards compatible.
+- Breaking changes to components such as updating the API or visual changes.
```

Comment:
I think we need to spend a bit more time figuring out what a breaking visual change consists of. That's probably outside the scope of this PR but we figure it out soon nonetheless.

Reply:
I guess we could look at creating a notes section to outline and detail what a visual change is?